src/OrderFormAcceptanceTests.Objects/Pages/PrintOrderSummary.cs

```diff
@@ -8,7 +8,7 @@ namespace OrderFormAcceptanceTests.Objects.Pages
 {
     public class PrintOrderSummary
     {
-        public By PrintableOrderSummary => CustomBy.DataTestId("printable-order-summary");
-        public By PrintPreview => By.Name("print-preview-app");
+        public By PrintableOrderSummary => CustomBy.DataTestId("preview-page-title");

```

Comment:
this isn't finding a thing on the print preview, it's finding an element on the original page

Reply:
Yeah the content of the main window is passed to a plug in in the print preview so you cant check for it being present.   So you cant check for content elements in the print preview.  

<embed id="plugin" type="application/x-google-chrome-pdf" src="chrome://print/1/0/print.pdf" stream-url="chrome://print/1/0/print.pdf" headers="" background-color="0xFF525659" top-toolbar-height="0" javascript="block" top-level-url="undefined">

So i think the best you can do is check the content in the main window that is being printed and then check that the print dialog is present?

Reply:
I'd look for this on the preview `[src='chrome://print/1/0/print.pdf']`

Reply:
With Fawads last change  the print summary was moved to its own tab. So this test now checks that that tab is present and has the correct content.